hygieia-jenkins-plugin/src/test/java/jenkins/plugins/hygieia/HygieiaPublisherTest.java

```diff
@@ -106,6 +106,10 @@ public HygieiaResponse publishDeployData(DeployDataCreateRequest request) {
             return hygieiaResponse;
         }
 
+        public RestCall.RestCallResponse getStageResponse(String url, String jenkinsUser, String token){
+            return  null;
```

Comment:
Is this method needed?

Reply:
@rvema  yeah, it is overriden method and should be implemented, added @Ovverride